apps/vite/src/components/calendar/CalendarGrid.tsx

```
@@ -1,39 +1,98 @@
+          return (
+            <button
+              key={day.calendarInfo.dates.bs.day.np}
+              className={cn(
+                "flex xl:aspect-square max-xl:min-h-[60px] p-3.5 bg-white  border-indigo-200 transition-all duration-300 hover:bg-gray-200/80 cursor-pointer relative",
+                isHoliday && "text-red-500",
+                isToday && " text-indigo-600"
+              )}
+              style={
+                index === 0
+                  ? {
+                      gridColumnStart: day.calendarInfo.days.codes.en!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)